apps/api/tsconfig.json

```diff
@@ -1,18 +1,15 @@
 {
+  "extends": "@recallnet/typescript-config/base.json",
   "compilerOptions": {
-    "target": "ES2020",
-    "module": "commonjs",
-    "lib": ["es2020"],
```

Comment:
Went ahead and allowed these to come from the shared config. This uses more modern settings, but it didn't cause many problems since you're already writing modern Typescript.